datascroller/scroller.py

```diff
@@ -282,7 +281,7 @@ def _display_string_rep_using_curses(self, screen, otherstring=None):
             time.sleep(3)
 
         screen.attroff(curses.color_pair(1))
-        curses.endwin() # NOTE: Still not sure about the persistance of screen
+        curses.endwin()  # NOTE: Still not sure about the persistance of screen
```

Comment:
I also don't think this comments doing anything for me or anybody else. Feel free to delete. Optional